### ISSUE TITLE
[FR] choose which log files a user can see

### ISSUE THREAD

**Issue by jonleverrier** [verbb/timber#3]
### What are you trying to do?

I'm doing some custom logging.

I would like to expose this data in the control panel using Timber for certain CP users to view. However, I don't want these CP users to view all log files. Just the ones I define.

### What's your proposed solution?

- Add a permission in user permissions that allows an administrator to choose what log files can be seen.
- Or, add a config option in `timber.php`

**Comment by engram-design** [comment]
Good idea, but this might be tricky in practice because not all the available log files are known at the time. We know the Craft ones (`web`, `console`, `queue` and `phperrors`), but as a lot of plugins also use log files (Verbb's in particular) we might have to figure out how to handle that.

**Comment by jonleverrier** [comment]
I'm sure you could think of something better, but how about something like this as a concept in timber.php

- `customLogListUserGroupId` (integer) - only cp users that belong to this user group id see the custom log selection 
- `customLogListSelection` (array) - an array of log names that you want to display for this user group. If null or not set it shows nothing to that user group. `["web", "mycustomlogfile"]` would only show log files for `web` and `mycustomlogfile`
